src/ng/directive/attrs.js

```diff
@@ -159,7 +159,7 @@
  *
  * @description
  *
- * This directive sets the `disabled` attribute on the element if the
+ * This directive sets the `disabled` attribute on a form element if the
```

Comment:
I think there are only these element that natively respond to disabled: option, input, textarea, button, select. Why not list them all?